tapdb/universe.go

```diff
@@ -425,9 +425,41 @@ func universeRegisterIssuance(ctx context.Context, dbTx BaseUniverseStore,
 func (b *BaseUniverseTree) FetchIssuanceProof(ctx context.Context,
 	universeKey universe.BaseKey) ([]*universe.IssuanceProof, error) {
 
-	// Depending on the universeKey, we'll either be fetching the details
-	// of a specific issuance, or all of the issuances for that minting
-	// outpoint.
+	var (
+		readTx = NewBaseUniverseReadTx()
+
+		err    error
+		proofs []*universe.IssuanceProof
+	)
+
+	err = b.db.ExecTx(ctx, &readTx, func(dbTx BaseUniverseStore) error {
```

Comment:
Should we stick with the `dbErr` naming here? I guess otherwise it's a bit confusing as the `err` will be overwritten. So I'd move the `var err error` inside the closure, then just have `dbErr` on the outside.